packages/gen-ai/frontend/src/app/AIAssets/AIAssetsModelsTab.tsx

```diff
@@ -2,23 +2,46 @@ import * as React from 'react';
 import { Bullseye, Content, ContentVariants, Spinner } from '@patternfly/react-core';
 import { useNavigate } from 'react-router-dom';
 import ModelsEmptyState from '~/app/EmptyStates/NoData';
-import useFetchLlamaModels from '~/app/hooks/useFetchLlamaModels';
 import { GenAiContext } from '~/app/context/GenAiContext';
+import useFetchLlamaModels from '~/app/hooks/useFetchLlamaModels';
+import { LlamaModel } from '~/app/types';
+import AIModelsTable from './components/AIModelsTable';
+import { AIModel } from './types';
+import useFetchAIModels from './hooks/useFetchAIModels';
 
 const AIAssetsModelsTab: React.FC = () => {
   const navigate = useNavigate();
   const { namespace } = React.useContext(GenAiContext);
-  const { data: models, loaded } = useFetchLlamaModels(namespace?.name);
+  const { data: models, loaded, error } = useFetchAIModels(namespace?.name);
+  const { data: playgroundModels, loaded: playgroundModelsLoaded } = useFetchLlamaModels(
+    namespace?.name,
+  );
+
+  // Determine which AI Assets models are available in the playground
+  const modelsWithPlaygroundStatus = React.useMemo(() => {
+    const playgroundModelIds = new Set(playgroundModels.map((model: LlamaModel) => model.id));
 
-  if (!loaded) {
+    return models.map((model: AIModel) => ({
+      ...model,
+      playgroundStatus: playgroundModelIds.has(model.model_name) ? 'available' : 'not-available',
+    }));
+  }, [models, playgroundModels]);
+
+  const handleTryInPlayground = (model: AIModel) => {
+    // Navigate to playground with the selected model
+    const playgroundUrl = `/gen-ai/playground/${namespace?.name}?model=${encodeURIComponent(model.model_name)}`;
```

Comment:
Based on the discussion here https://redhat-internal.slack.com/archives/C099MEPGF43/p1757526668691629?thread_ts=1757523105.673619&cid=C099MEPGF43. The preferred way could be using `{state: {modelName: model.model_name}}` to pass the data between 2 pages because we don't want to bookmark the data a lot. We only want to use it for selection when navigates it.
In the Chatbot, it only needs to create `const location = useLocation()`, read it from `location.state` and select it if it exists.